<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IC</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QTabWidget" name="toolstab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="ICTab_FilterRack" name="tab_filterrack">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Filter Rack</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="sca_filterrack">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="filterrack_contents">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>242</width>
             <height>472</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="pb_add_filter">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources/ic_resources.qrc">
                <normaloff>:/icon/add</normaloff>:/icon/add</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="ICPlaybackFrame" name="frm_playback">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="pb_play">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources/ic_resources.qrc">
           <normaloff>:/icon/play</normaloff>:/icon/play</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pb_loop">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources/ic_resources.qrc">
           <normaloff>:/icon/refresh</normaloff>:/icon/refresh</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lbl_current">
         <property name="text">
          <string>00:00:000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QScrollBar" name="scb_pos">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="lbl_total">
         <property name="text">
          <string>00:00:000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTabWidget" name="maintab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="ICTab_Preview" name="tab_preview">
       <attribute name="title">
        <string>Preview</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel_ResizablePixmap" name="lbl_preview">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="locale">
    <locale language="Portuguese" country="Brazil"/>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_preview">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Preview</string>
    </property>
    <widget class="QMenu" name="menu_preview_source">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Preview source</string>
     </property>
     <addaction name="actn_raw"/>
     <addaction name="actn_post_filter"/>
     <addaction name="actn_post_analysis"/>
    </widget>
    <addaction name="menu_preview_source"/>
   </widget>
   <widget class="QMenu" name="menu_video">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actn_load_input_plugin"/>
    <addaction name="actn_close_input_plugin"/>
    <addaction name="separator"/>
    <addaction name="actn_load_analysis_plugin"/>
    <addaction name="actn_close_analysis_plugin"/>
   </widget>
   <widget class="QMenu" name="menu_analysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actn_new_analysis"/>
    <addaction name="actn_open"/>
    <addaction name="separator"/>
    <addaction name="actn_save"/>
    <addaction name="actn_save_as"/>
    <addaction name="separator"/>
    <addaction name="actn_show_settings"/>
    <addaction name="separator"/>
    <addaction name="actn_exit"/>
   </widget>
   <widget class="QMenu" name="menu_console">
    <property name="title">
     <string>Console</string>
    </property>
    <addaction name="actn_show_console"/>
   </widget>
   <addaction name="menu_analysis"/>
   <addaction name="menu_video"/>
   <addaction name="menu_preview"/>
   <addaction name="menu_console"/>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actn_load_input_plugin"/>
   <addaction name="actn_close_input_plugin"/>
   <addaction name="separator"/>
   <addaction name="actn_load_analysis_plugin"/>
   <addaction name="actn_close_analysis_plugin"/>
   <addaction name="separator"/>
   <addaction name="actn_add_filter"/>
   <addaction name="separator"/>
  </widget>
  <action name="actn_raw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/tv</normaloff>:/icon/tv</iconset>
   </property>
   <property name="text">
    <string>Raw</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_post_filter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/tv</normaloff>:/icon/tv</iconset>
   </property>
   <property name="text">
    <string>Post-Filter</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_post_analysis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/tv</normaloff>:/icon/tv</iconset>
   </property>
   <property name="text">
    <string>Post-Analysis</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_load_input_plugin">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/load</normaloff>:/icon/load</iconset>
   </property>
   <property name="text">
    <string>Load Input Plugin</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_close_input_plugin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/close</normaloff>:/icon/close</iconset>
   </property>
   <property name="text">
    <string>Close Input Plugin</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_new_analysis">
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/new_document</normaloff>:/icon/new_document</iconset>
   </property>
   <property name="text">
    <string>New Analysis</string>
   </property>
   <property name="toolTip">
    <string>Create a new Analysis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_open">
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/folder</normaloff>:/icon/folder</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open an analysis file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/save</normaloff>:/icon/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current analysis to a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_save_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actn_exit">
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/exit</normaloff>:/icon/exit</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit from application</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_add_filter">
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/add</normaloff>:/icon/add</iconset>
   </property>
   <property name="text">
    <string>Add Filter</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_clearfilterrack">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actn_show_settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/wrench</normaloff>:/icon/wrench</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Edit Analysis Settings</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_load_analysis_plugin">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/magnifying_glass</normaloff>:/icon/magnifying_glass</iconset>
   </property>
   <property name="text">
    <string>Load Analysis Plugin</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_close_analysis_plugin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/close</normaloff>:/icon/close</iconset>
   </property>
   <property name="text">
    <string>Close Analysis Plugin</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actn_show_console">
   <property name="icon">
    <iconset resource="resources/ic_resources.qrc">
     <normaloff>:/icon/console</normaloff>:/icon/console</iconset>
   </property>
   <property name="text">
    <string>Show console</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICPlaybackFrame</class>
   <extends>QFrame</extends>
   <header>playback_frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ICTab_Preview</class>
   <extends>QWidget</extends>
   <header>tab_preview</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLabel_ResizablePixmap</class>
   <extends>QLabel</extends>
   <header>resizable_pixmap_label</header>
  </customwidget>
  <customwidget>
   <class>ICTab_FilterRack</class>
   <extends>QWidget</extends>
   <header>tab_filter_rack</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/ic_resources.qrc"/>
 </resources>
 <connections/>
</ui>
